packages/rum/test/e2e/react/webpack.config.js

```diff
@@ -37,9 +42,23 @@ module.exports = {
     rules: [
       {
         test: /.jsx?$/,
-        include: [__dirname, path.resolve(__dirname)],
         use: {
-          loader: 'babel-loader'
+          loader: 'babel-loader',
+          options: {
+            presets: [
+              [
+                '@babel/preset-env',
```

Comment:
does it work without preset-react?

Reply:
You dont need to pass these explicity here. We have babel.config.js for test directory which has these options by default.